### PR TITLE
Upgraded Lion and lit

### PR DESCRIPTION
**Pull request by vishalshrm539**
* Upgraded Lion and lit.
* Fixed other issues.

### CODE REVIEW

File: src/samples/SimplePortal/SimpleSideBar/_lion_core.code-search
Comment (tumms2021389) [inline src/samples/SimplePortal/SimpleSideBar/_lion_core.code-search:1]: What is this file? Do we really need it? if not please add it to gitignore and delete this file

File: src/samples/SimplePortal/SimpleSideBar/bootstrap-styles.ts
Comment (tumms2021389) [inline src/samples/SimplePortal/SimpleSideBar/bootstrap-styles.ts:1]: This file already present in BridgeBase folder, can't we use same file?

File: tsconfig.json [hunk 1]
```diff
@@ -20,10 +20,9 @@
     "isolatedModules": true,
     "useDefineForClassFields": false,
     "strict": true,
-    "sourceMap": true,
```
Comment (tumms2021389) [inline tsconfig.json:23]: Any specific reason to remove sourceMap?

Comment (vishalshrm539) [reply tsconfig.json:23]: It was a duplicate.

File: src/samples/SimplePortal/SimpleSideBar/bootstrap-styles.ts
Comment (vishalshrm539) [reply src/samples/SimplePortal/SimpleSideBar/bootstrap-styles.ts:1]: Removed.

File: src/samples/SimplePortal/SimpleSideBar/_lion_core.code-search
Comment (vishalshrm539) [reply src/samples/SimplePortal/SimpleSideBar/_lion_core.code-search:1]: Probably genearted by lion, will ignore it moving forward.

**Review by tumms2021389 — APPROVED** [review]
Looks good to me

**Pull request merged**